src/Refactoring-Core/RBInlineTemporaryRefactoring.class.st

```diff
@@ -38,55 +38,70 @@ RBInlineTemporaryRefactoring class >> model: aRBSmalltalk inline: anInterval fro
 ]
 
 { #category : 'preconditions' }
-RBInlineTemporaryRefactoring >> applicabilityPreconditions [
+RBInlineTemporaryRefactoring >> applicabilityPreconditions [ 
 
+	| validator |
+	
 	^ {
-		  (RBCondition definesSelector: selector in: class).
-		  (RBCondition withBlock: [
-			   self checkSelectedInterval.
-			   true ]) }
+		(ReDefinesSelectorsCondition new definesSelectors: (Array with: selector) in: class).
+		
+		(ReBlockCondition new 
+			block: [
+				self checkSelectedInterval.
```

Comment:
It would be nice if we could inline this method's call, since the method contains two conditions and they might be encapsulated.